cyclus_input/reactors/Arkansas_Nuclear_1.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" standalone="no" ?>
+
+<simulation>
```

Comment:
I made a comment in a previous review of this PR about these simulation tags. They should not be in the individual reactor xml files.